### DISCUSSION TITLE
drag and drop question

### DISCUSSION THREAD

**Discussion by SmackdabDevOps** [xyflow/xyflow#3726] [Q&A]
[https://www.loom.com/share/c90e5385e1634195944e12b85a477eda](url)

I have implemented a hierarchy using ReactFlow and have attached a video. Currently, when clicking on both the block and outside the block, the user can drag the entire hierarchy. However, I want to disable dragging specifically for the textarea. This is important because users are currently unable to select text within the textarea while the entire hierarchy is draggable.

**Comment by bcakmakoglu** [comment]
Set the class name `nodrag` onto your textarea element to stop it from being an element that can initiate dragging a node.

```tsx
<textarea className="nodrag" ... />
```
